implementation/observation-otel-bridge/src/main/java/io/smallrye/opentelemetry/instrumentation/observation/handler/AbstractTracingObservationHandler.java

```diff
@@ -108,16 +110,37 @@ protected Span getParentSpan(T context) {
         return null;
     }
 
+    @SuppressWarnings("unchecked")
     protected void tagSpan(T context, Span span) {
+        final Attributes highCardAttributes = context.get(HIGH_CARD_ATTRIBUTES);
+        setOtelAttributes(span, highCardAttributes);
+
+        final Attributes lowCardAttributes = context.get(LOW_CARD_ATTRIBUTES);
+        setOtelAttributes(span, lowCardAttributes);
+
         for (KeyValue keyValue : context.getAllKeyValues()) {
```

Comment:
Can it contain keys from `HIGH_CARD_ATTRIBUTES`and `LOW_CARD_ATTRIBUTES`?

Reply:
yes, but stored in 2 different structures.